src/command.c

```diff
@@ -627,6 +628,32 @@ int cmd_auth(struct command *cmd, struct redis_data *data)
     return CORVUS_OK;
 }
 
+int cmd_time(struct command *cmd)
+{
+	struct timeval tm;
+	int ret = gettimeofday(&tm, NULL);
+	if (ret) {
+		return CORVUS_ERR;
+	}
+	char time_fmt[100];
+	char time_sec[15];
+	char time_us[10];
+	snprintf(time_sec, sizeof(time_sec), "%d", tm.tv_sec);
+	snprintf(time_us, sizeof(time_us), "%d", tm.tv_usec);
```

Comment:
In Linux `tv_sec` and `tv_usec` are all `long`, it would be better to use `%ld`  instead of `%d`. And `time_sec` and `time_us` are not large enough to hold 8 bytes integer.
 ```c
char time_sec[21], time_us[21];
int sec_len = snprintf(time_sec, 20, "%ld", tm.tv_sec);
int usec_len = snprintf(time_us, 20, "%ld", tm.tv_usec);
```